packages/atomic/src/components/facets-v1/facet-date-input/facet-date-input.tsx

```diff
@@ -0,0 +1,128 @@
+import {Component, h, State, Prop, Event, EventEmitter} from '@stencil/core';
+import {DateFilter, DateFilterState, buildDateRange} from '@coveo/headless';
+import {Bindings} from '../../../utils/initialization-utils';
+import dayjs from 'dayjs';
+
+/**
+ * Internal component made to be integrated in a TimeframeFacet.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-facet-date-input',
+  shadow: false,
+})
+export class FacetDateInput {
```

Comment:
Since this contains two inputs, I think it should be named something like `FacetDateRangePicker`.

Reply:
I see the name as a user "inputting" a custom range, not as the element. Its like the FacetNumberInput. Could change the name for both 🤷